include/experimental/kernel/gemm/impl/int4_dequantize_kslicing_xe.hpp

```diff
@@ -771,8 +299,13 @@ class gemm_universal_t<dispatch_policy_int4_dequantize_kslicing<group_swizzle_,
     __XETLA_API static constexpr uint32_t get_slm_size() {
         constexpr uint32_t size = gemm_slm_size * num_local_kslicing
                 + kslicing_slm_size + epilogue_slm_size * num_local_kslicing;
-        static_assert(size <= (128 * 1024),
-                "The local memory size should be less than 128KB!");
+        if constexpr (arch_tag == gpu_arch::Dg2) {
```

Comment:
as a 'xxx_xe.hpp', should arch_tag be fixed one in this file like L128 does?

Reply:
xxx_xe.hpp I think it is not a  good name.
But it better synchronization with xetla innersource